UI/window-basic-auto-config.cpp

```diff
@@ -584,6 +584,13 @@ void AutoConfigStreamPage::UpdateKeyLink()
 		text += QTStr(
 			"Basic.AutoConfig.StreamPage.StreamKey.LinkToSite");
 		text += "</a>";
+	} else if (serviceName.startsWith("YouStreamer")) {
```

Comment:
This should be done using an `==` comparison, `startsWith` is for services with multiple entries (eg RTMP and FTL).